src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -158,8 +158,18 @@ public partial class NewAzureRmVmssConfigCommand : Microsoft.Azure.Commands.Reso
 
         [Parameter(
             Mandatory = false,
-            ValueFromPipelineByPropertyName = false)]
-        public SwitchParameter AssignIdentity { get; set; }
```

Comment:
Removing this is a breakign change.  You need to add a new parameter set for additiinal user assigned identy types.

Reply:
This parameter is kept in Manual/NewAzureRmVmssConfigCommnad.cs file.